api_bonus.yaml

```diff
@@ -0,0 +1,144 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Bonus API
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  "/bonus/vacanze/request":
+    post:
+      operationId: startBonusEligibilityCheck
+      summary: Start bonus eligibility check (ISEE)
+      responses:
+        '202':
+          description: Request accepted.
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          headers:
+            Location:
+              type: string
+              description: |-
+                Location (URL) of created request resource.
+                A GET request to this URL returns the request status and details.
+          examples:
+            application/json:
+              id: 01BX9NSMKVXXS5PSP2FATZMYYY
+
+    get:
```

Comment:
shouldn't this have the eligibility request ID in the path?

```
GET /bonus/vacanze/eligibility/01BX9NSMKVXXS5PSP2FATZMYYY
```

Reply:
not if the id equals the user fiscal code. since we want to have one single eligibility request across devices and user's sessions, we can use the CF as the id

Reply:
(the CF is not actually needed as it's implicitly defined by the session token)

Anyway I would still use a specific task ID for each eligibility request as it makes it easier to troubleshoot specific requests and track them in logs (vs all app instances querying the same URL).
The invariant of only one task should be enforced by `startBonusEligibilityCheck` (returning 409 CONFLICT in case a running eligibility task already exist)